### PR TITLE
Get selected values from dropdowns

### PR DESCRIPTION
**Pull request by fionaphillips**
Resolves [CG-40](https://unosquarebelfast.atlassian.net/secure/RapidBoard.jspa?rapidView=12&projectKey=CG&modal=detail&selectedIssue=CG-40)

Screenshot 2019-11-18 at 10 48 16 (2)

### CODE REVIEW

File: src/components/GenerateCVButton/container.js [hunk 1]
```diff
@@ -1,18 +1,43 @@
 import React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { PropTypes as PT } from 'prop-types';
 
-export default Wrapped =>
+import { getEmployee, getTemplate } from '../../store/reducers/selected-options';
```
Comment (girwin90) [inline src/components/GenerateCVButton/container.js]: Can we use an alias here?

File: src/components/common/Select/container.js [hunk 2]
```diff
@@ -39,3 +65,19 @@ export default Wrapped =>
       );
     }
   };
+
+function mapDispatchToProps(dispatch) {
```
Comment (girwin90) [inline src/components/common/Select/container.js]: can we make this an anonymous function just for consistency across the app?

File: src/components/common/Select/container.js [hunk 3]
```diff
@@ -39,3 +65,19 @@ export default Wrapped =>
...
+}
+
+export default compose(
+  connect(
+    null,
+    mapDispatchToProps,
+  ),
+  SelectContainer,
+);
+/* eslint-enable react/destructuring-assignment */
```
Comment (girwin90) [inline src/components/common/Select/container.js]: is this needed? Can we not do this in an eslint file?

File: src/components/constants.js [hunk 4]
```diff
@@ -1,6 +1,9 @@
-const buttonTypes = {
+export const buttonTypes = {
   submit: 'submit',
   button: 'button',
 };
 
-export default buttonTypes;
+export const selectFields = {
+  employee: 'selectComponentEmployee',
```
Comment (girwin90) [inline src/components/constants.js]: Capital letters for keys in a constant object.

File: src/pages/Selector/container.jsx [hunk 5]
```diff
@@ -7,13 +7,14 @@ const SelectorContainer = Wrapped =>
     state = { employees: [], templates: [] };
 
     componentDidMount() {
+      /* eslint-disable no-shadow */
```
Comment (girwin90) [inline src/pages/Selector/container.jsx]: Can we do this in an eslint rc file or something? Looks just a bit messy in my opinion

File: src/store/actions/action-types.js [hunk 6]
```diff
@@ -0,0 +1,2 @@
+export const SELECT_EMPLOYEE = 'SELECT_EMPLOYEE';
```
Comment (girwin90) [inline src/store/actions/action-types.js:1]: Nice!

File: src/store/reducers/index.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+const EMPLOYEES = 'EMPLOYEES';
+const TEMPLATES = 'TEMPLATES';
+
+const rootReducer = combineReducers({
+  [EMPLOYEES]: selectedOptionsReducer,
+  [TEMPLATES]: selectedOptionsReducer,
+});
+export default rootReducer;
+
+export const getEmployee = store => FromSelectedOptions.getEmployee(store[EMPLOYEES]);
```
Comment (girwin90) [inline src/store/reducers/index.js]: Are these selectors? Maybe they can be elsewhere?

**Review by girwin90 — CHANGES_REQUESTED** [review]

**Review by BrownMark — APPROVED** [review]
Looks great, all coming together pretty well :)

**Pull request merged**